config/dev.json

```diff
@@ -6,5 +6,6 @@
   },
   "img": {
     "driver": "local"
-  }
+  },
+  "CUSTOMS_SERVER_URL": "none"
```

Comment:
this should be something like:

```
"customsUrl": "none"
```

these need to be the same as the keys in lib/config.js, not env variables.